Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+//! Solution to Week-01/Day-01.
+//!
+//! Convert years to days.
+
+use num::BigUint;
+
+/// Number of days in a year.
+pub const YEAR_TO_DAYS: u32 = 365;
+
+/// Returns the number days in a given number of years. A year has [YEAR_TO_DAYS] days. Leap years are not supported.
+///
+/// # Argument
+///
+/// * `years` - the number of years to convert to days
+///
+/// # Example
+///
+/// ```
+/// # use num::BigUint;
+/// # use convert_ages_to_days_brainrake::years_to_days;
+/// let days = years_to_days(BigUint::from(1_u32));
+/// assert_eq!(days, BigUint::from(365_u32));
+/// ```
+pub fn years_to_days(years: BigUint) -> BigUint {
+    years * YEAR_TO_DAYS
+}
+
+
+#[cfg(test)]
+mod tests {
+    use crate::years_to_days;
+    use num::BigUint;
+    use std::str::FromStr;
+
+    #[test]
+    fn test_0() {
```

Comment:
Same applies here as in [this PR comment](https://github.com/Smorci/100-Days-Of-Rust/pull/4/files#r1498273676)